pkgs/by-name/co/cosmic-ext-calculator/package.nix

```diff
@@ -0,0 +1,78 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  just,
+  pkg-config,
+  makeBinaryWrapper,
+  libxkbcommon,
+  libGL,
+  xorg,
+  wayland,
+  vulkan-loader,
+  cosmic-icons,
+  nix-update-script,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "cosmic-ext-calculator";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "cosmic-utils";
+    repo = "calculator";
+    tag = version;
+    hash = "sha256-UO3JKPsztptNEFAxolRui9FxtCsTRgpXhHH242i9b6E=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-a4IOmCWKX8RR8xeKS6wdsTlNyTr91B93bYMDx/+/+04=";
+
+  nativeBuildInputs = [
+    just
+    pkg-config
+    makeBinaryWrapper
+  ];
+
+  buildInputs = [
+    libGL
+    libxkbcommon
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXi
+    xorg.libxcb
+    wayland
+    vulkan-loader
+  ];
+
+  dontUseJustBuild = true;
+  dontUseJustCheck = true;
+
+  justFlags = [
+    "--set"
+    "prefix"
+    (placeholder "out")
+    "--set"
+    "bin-src"
+    "target/${stdenv.hostPlatform.rust.cargoShortTarget}/release/${meta.mainProgram}"
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/${meta.mainProgram} \
+      --suffix XDG_DATA_DIRS : "${cosmic-icons}/share" \
+      --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath buildInputs}"
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    changelog = "https://github.com/cosmic-utils/calculator/releases/tag/${src.tag}";
```

Comment:
```suggestion
    changelog = "https://github.com/cosmic-utils/calculator/releases/tag/${version}";
```

Reply:
It is better to follow `src.tag` since the tag can change and not be just the version

Reply:
I asked about such cases a few days ago in the matrix room and got the answer that `version` is prefered over `src.tag`

> src could be overridden with something that has no tag. that probably doesn't matter a whole lot for the changelog, but still worth preferring version

Reply:
Makes sense, `src.tag or version` could avoid this problem

Reply:
Done!